TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not settings.get('current_file', False):
```

Comment:
Logic appears to be reversed here. Change to `if args.get('current_file', False):`.

It also seems that this argument should have the highest priority, so all the following checks should not take place when this is `True`.